extensions/ql-vscode/src/remote-queries/export-results.ts

```diff
@@ -95,3 +97,32 @@ async function exportResultsToGist(
     }
   }
 }
+
+/**
+ * Converts the results of a remote query to markdown and saves the files locally
+ * in the query directory (i.e. the workspace directory where query results and logs are saved).
+ */
+async function exportResultsToLocalMarkdown(
+  queryHistoryManager: QueryHistoryManager,
+  queryHistoryItem: QueryHistoryInfo,
+  query: RemoteQuery,
+  analysesResults: AnalysisResults[]
+) {
+  const markdownFiles = generateMarkdown(query, analysesResults, 'local');
+  const basePath = await queryHistoryManager.getQueryHistoryItemDirectory(
+    queryHistoryItem
+  );
+  for (const markdownFile of markdownFiles) {
+    const filePath = path.join(basePath, `${markdownFile.fileName}.md`);
+    await fs.writeFile(filePath, markdownFile.content.join('\n'), 'utf8');
+  }
+  const shouldOpenSummary = await showInformationMessageWithAction(
+    `Variant analysis results exported to \"${basePath}\".`,
+    'Open exported results'
+  );
+  if (shouldOpenSummary) {
```

Comment:
Hmm, that's a good point. You can currently access the results files in the "query directory" from here:
![image](https://user-images.githubusercontent.com/42641846/168327018-1d60336c-fa00-4ae6-b0be-5eaf7ba5b9ab.png)

That isn't at all obvious though 😅 Perhaps we could put all the results in a "exported results" subdirectory of the query directory, and then open that directory instead of just opening the summary file?

Reply:
I think that would work. Could you do both? ie- open the markdown file as an editor as well as open the exported results directory? I think it's nice to see the prettified markdown version of the results.